packages/react/package.json

```diff
@@ -24,6 +24,7 @@
   },
   "dependencies": {
     "@mintbase-js/wallet": "0.6.1-update-packages-24058ee.0",
+    "@mintbase-js/data": "0.6.1-update-packages-24058ee.0",
```

Comment:
Just a thought, we could use this PR to find and replace all of `0.6.1-update-packages-24058ee.0` with with `^0.6.1`